Maestro/subscriptions.json

```diff
@@ -299,6 +305,13 @@
         }
       }
     },
+    // Mirror corefx changes into VSO
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/**/*",
```

Comment:
Oops, didn't notice the trigger had both versions and corefx in it. Updated all the paths.